src/run-tests.xml

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE MudletPackage>
 <MudletPackage version="1.001">
-  <TriggerPackage />
-  <TimerPackage />
-  <AliasPackage>
-    <Alias isActive="yes" isFolder="no">
-      <name>Run Tests</name>
-      <script>-- Alias to run busted tests, which should be sufficient
+	<TriggerPackage />
```

Comment:
What is with the spacing changes?

Reply:
We changed the XML serialised a while back to pugixml

Reply:
Okay...